serverless/serverless-release-notes.adoc

```diff
@@ -25,4 +27,4 @@ include::modules/serverless-rn-1-10-0.adoc[leveloffset=+1]
 * For details about the latest Knative Serving Operator release, see the link:https://github.com/knative/serving-operator/releases[Knative Serving Operator releases page].
 * For details about the latest Knative CLI release, see the link:https://github.com/knative/client/releases[Knative CLI releases page].
 * For details about the latest Knative Eventing release, see the link:https://github.com/knative/eventing/releases[Knative Eventing releases page].
-* link:https://openshift-knative.github.io/docs/docs/functions/about-functions.html[{ServerlessProductName} Functions Developer Preview documentation].
+* link:https://github.com/boson-project/func/releases[Functions releases page].
```

Comment:
Did you get support to +1 this GitHub link?

Reply:
I'll remove this link for now and make an update for the others

Reply:
Changed this to a note at the start of the doc